Modules/Sources/Yosemite/Model/Storage/PersistedProductVariation+Conversions.swift

```diff
@@ -36,19 +36,55 @@ extension PersistedProductVariation {
             stockStatusKey: stockStatusKey
         )
     }
+
+    func toPOSProductVariation(db: GRDBDatabaseConnection) throws -> POSProductVariation {
+        let image = try db.read { db in
+            return try request(for: PersistedProductVariation.image).fetchOne(db)
+        }
+        let attributes = try db.read { db in
+            return try request(for: PersistedProductVariation.attributes).fetchAll(db)
+        }
```

Comment:
Similar to `PersistedProduct`, could potentially combine these two into one `db.read`.